webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/DSRemoteWebKnossosClient.scala

```diff
@@ -80,14 +80,14 @@ class DSRemoteWebKnossosClient @Inject()(
       .silent
       .put(dataSources)
 
-  def validateDataSourceUpload(info: ReserveUploadInformation, userTokenOpt: Option[String]): Fox[_] =
+  def validateDataSourceUpload(info: ReserveUploadInformation, userTokenOpt: Option[String]): Fox[Unit] =
     for {
-      userToken <- option2Fox(userTokenOpt) ?~> "initialTeams.noUserToken"
-    } yield
-      rpc(s"$webKnossosUri/api/datastores/$dataStoreName/verifyUpload")
+      userToken <- option2Fox(userTokenOpt) ?~> "validateDataSourceUpload.noUserToken"
+      _ <- rpc(s"$webKnossosUri/api/datastores/$dataStoreName/verifyUpload")
         .addQueryString("key" -> dataStoreKey)
         .addQueryString("token" -> userToken)
         .post(info)
+    } yield ()
```

Comment:
This is a bug introduced in #5831 – the result of the rpc was not unpacked (and thus a Fox[Fox[WSResponse]] returned). Now I unpack it, so if the outer *or* the inner fox fails, the return value is a Failure as well